README.md

```diff
@@ -53,7 +61,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n) because the graph of this problem will grow linearly as the length of the path increases.
```

Comment:
# 1

I don't understand what "the graph...will increase means".

Basically, it it O(n) because you have to keep at least half the input on the stack while processing.